src/apis/avm/api.ts

```diff
@@ -1278,16 +1291,83 @@ export class AVMAPI extends JRPCAPI {
     } else {
       amnt = amount;
     }
+    if(typeof memo !== 'string') {
+      m = bintools.cb58Encode(memo);
+    } else {
+      m = memo;
+    }
 
     const params:any = {
-      username,
-      password,
+      username: username,
+      password: password,
       assetID: asset,
       amount: amnt.toString(10),
-      to,
-      from,
+      to: to,
+      from: from,
+      changeAddr: changeAddr,
```

Comment:
we can't add memo and changeAddress in by default. They only are added into the params if they're defined, and since they're optional, that's not guaranteed.